tests/testthat/_snaps/without_financial.md

```diff
@@ -3,18 +3,18 @@
     Code
       without_financial
     Output
-      # A tibble: 58 x 5
-         company_name  xctr_risk_category benchmark     product_name tilt_sector
-         <chr>         <chr>              <chr>         <chr>        <chr>      
-       1 peter peasant high               all           car          D          
-       2 peter peasant high               unit          car          D          
-       3 peter peasant medium             tilt_sec      car          D          
-       4 peter peasant medium             unit_tilt_sec car          D          
-       5 peter peasant low                isic_sec      car          D          
-       6 peter peasant low                unit_isic_sec car          D          
-       7 peter         high               all           banana       A          
-       8 peter         high               unit          banana       A          
-       9 peter         medium             tilt_sec      banana       A          
-      10 peter         medium             unit_tilt_sec banana       A          
-      # i 48 more rows
+      # A tibble: 114 x 6
+         company_name postcode xctr_risk_category benchmark   product_name tilt_sector
+         <chr>           <int> <chr>              <chr>       <chr>        <chr>      
+       1 bruno           27568 high               all         car          D          
+       2 bruno           27568 high               all         steel        C          
+       3 bruno           27568 medium             unit        banana       B          
+       4 bruno           27568 medium             tilt_sec    banana       B          
+       5 bruno           27568 high               unit_tilt_~ banana       B          
+       6 bruno           27568 medium             isic_sec    banana       B          
+       7 bruno           27568 high               unit_isic_~ banana       B          
+       8 bruno           27568 medium             unit        car          D          
+       9 bruno           27568 high               tilt_sec    car          D          
+      10 bruno           27568 medium             unit_tilt_~ car          D          
```

Comment:
FYI we're departing from the truncated version of the benchamaks, e.g. not "tilt_sec" but "tilt_sector". 

This may be one of the changes you plan to adapt to new data, but in case you need it, here are some references:


* You can see that in action in the examples of tiltIndicator:

https://2degreesinvesting.github.io/tiltIndicator/reference/emissions_profile.html#ref-examples

```r
...
both |> unnest_product()
#> # A tibble: 456 × 7
#>    companies_id        grouped_by       risk_category profile_ranking clustered 
#>    <chr>               <chr>            <chr>                   <dbl> <chr>     
#>  1 antimonarchy_canine all              high                        1 tent      
#>  2 antimonarchy_canine isic_4digit      high                        1 tent      
#>  3 antimonarchy_canine tilt_sector      high                        1 tent      
#>  4 antimonarchy_canine unit             high                        1 tent      
#>  5 antimonarchy_canine unit_isic_4digit high                        1 tent      
#>  6 antimonarchy_canine unit_tilt_sector high                        1 tent      
#>  7 celestial_lovebird  all              high                        1 table hir…
#>  8 celestial_lovebird  isic_4digit      high                        1 table hir…
#>  9 celestial_lovebird  tilt_sector      high                        1 table hir…
#> 10 celestial_lovebird  unit             high                        1 table hir…
#> # ℹ 446 more rows
#
```

* And the changelog:

https://2degreesinvesting.github.io/tiltIndicator/news/index.html#tiltindicator-0009095

> The values of `grouped_by` are now less surprising (see related principle) ([#601](https://github.com/2DegreesInvesting/tiltIndicator/issues/601)). They now simply refer to the full name of the corresponding columns in the “co2” dataset (products or inputs) passed to the `emissions_profile*()` functions. For example, passing a column `products$tilt_sector` now yields the value “tilt_sector” in `group_by` – which before got cropped to “tilt_sec”.

Reply:
Cool !! I'll add the link to your comment to my ticket for my toy data modification. Thanks :)